android/buildSrc/src/main/kotlin/Config.kt

```diff
@@ -3,9 +3,9 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 object Config {
-    const val compileSdkVersion = 33
+    const val compileSdkVersion = 34
     const val buildToolsVersion = "30.0.3"
     const val minSdkVersion = 26
-    const val targetSdkVersion = 33
+    const val targetSdkVersion = 34
```

Comment:
There is one more thing to to 
https://developer.android.com/about/versions/14/behavior-changes-14#fgs-types. 

In the daemon android manifest you need to set a type: https://developer.android.com/about/versions/14/changes/fgs-types-required